db/PostgreSQL/seederPostgre.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-unused-vars */
+const Faker = require('faker');
+const { Client } = require('pg');
+
+const client = new Client({
+  database: 'reviews',
+  user: 'alexk',
+  password: '',
+});
+
+const select = 'select * from reviews';
+// Formatted date maker
+const dateFormatter = () => {
+  const date = Faker.date.past().toString();
+  const dateFormatted = `${date.substring(4, 10)},${date.substring(10, 15)}`;
+  return dateFormatted;
+};
+// Weighted rating generator function  - squeezed
+const ratingGenerator = () => {
+  const randomNum = Math.random() * 10;
+  if (randomNum > 0 && randomNum < 5) {
+    return 5;
+  } if (randomNum > 5 && randomNum < 6.5) {
+    return 4;
+  } if (randomNum > 6.5 && randomNum < 8) {
+    return 3;
+  } if (randomNum > 8 && randomNum < 9) {
+    return 2;
+  }
+  return 1;
+};
+
+// seed!
+let counter = 0;
+let t0 = Date;
+const insert = 'insert into reviews (dbId, urlId, username, datePosted,imageUrl,avatarImgUrl,text,rating,itemForSale) select * from unnest ($1::int[], $2::int[], $3::text[], $4::date[], $5::text[], $6::text[], $7::text[], $8::real[], $9::text[])';
+let cycle = 100;
+let values = Array;
+let quantityOfReviews = Number;
+let currentPictureId = Number;
+let i = Number;
+let j = Number;
+
+client.connect()
+  .then(async() => {
+    for (let x = 0; x < cycle; x += 1) {
+      t0 = new Date;
+      console.log(counter);
+      values = [[],[],[],[],[],[],[],[],[]];
```

Comment:
Interesting. So you have an array where you are pushing the values of each field (userName, avatarURL) into an array and then adding that array to the database. I'm just making a note here because I am still researching Postgres.